Header header
geometry_msgs/Vector3 des_p
geometry_msgs/Vector3 u_p_p
geometry_msgs/Vector3 u_p_i
geometry_msgs/Vector3 u_p
geometry_msgs/Vector3 des_v
geometry_msgs/Vector3 u_v_p
geometry_msgs/Vector3 u_v_i
geometry_msgs/Vector3 u_v
geometry_msgs/Vector3 k_p_p
geometry_msgs/Vector3 k_p_i
geometry_msgs/Vector3 k_v_p
geometry_msgs/Vector3 k_v_i
